mypy/nodes.py

```diff
@@ -1648,9 +1647,10 @@ def accept(self, visitor: NodeVisitor[T]) -> T:
 #
 # If T is contravariant in Foo[T], Foo[object] is a subtype of
 # Foo[int], but not vice versa.
+CONTRAVARIANT = -1  # type: int
 INVARIANT = 0  # type: int
 COVARIANT = 1  # type: int
-CONTRAVARIANT = 2  # type: int
+# TODO: SELF_VARIANCE = 2  # type: int
```

Comment:
What do you think? I am not sure what this is, For now simple INVARIANT seem to suffice